ganttproject/src/main/java/net/sourceforge/ganttproject/GanttProject.java

```diff
@@ -880,4 +895,22 @@ public void refresh() {
     super.repaint();
   }
 
+  // TODO: Accumulate changes instead of sending it every time.
+  private Unit sendProjectStateLogs() {
+    gpLogger.debug("Sending project state logs");
+    try {
+      var logs = myProjectDatabase.fetchLogRecords(myLastReceivedXlogId.get() + 1, 10);
```

Comment:
It seems that every new task send also the xlogs of the previously created task.